apps/web/src/utils/skillUtils.ts

```diff
@@ -214,3 +214,19 @@ export const getExperiencesSkillIds = (experiences: Experience[]): string[] => {
 
   return deDupedIdCollection;
 };
+
+/**
+ * Parse a comma-separated list into an array of strings
+ * @param {string | null | undefined} value A single string, representing a comma-separated list. Or, may be an empty list or undefined.
+ * @returns {string[] | null}
+ */
+export const parseKeywords = (
+  value: string | null | undefined,
+): string[] | null => {
+  return value?.trim()
+    ? value
+        .split(".")
```

Comment:
That splits on a "." rather than "," as mentioned in the function description and form label using the function

Reply:
Oops! 😵